app/src/main/java/org/simple/clinic/overdue/AppointmentConfig.kt

```diff
@@ -0,0 +1,3 @@
+package org.simple.clinic.overdue
+
+data class AppointmentConfig(val highlightHighRiskPatients: Boolean)
```

Comment:
Do you feel we should move the age limit for calculation of high risk patients here as well?

Reply:
Yes, from a comment I posted to another conversation:

> Makes sense. I'll also add this to the config when I have more details about how we're categorizing high risk patients.